package.json

```diff
@@ -1,9 +1,35 @@
 {
-  "name": "ASP.NET",
-  "version": "0.0.0",
-  "devDependencies": {
-    "gulp": "3.8.11",
-    "gulp-rename": "1.2.2",
-    "gulp-uglify": "1.2.0"
-  }
+    "name": "jquery-validation-unobtrusive",
```

Comment:
For reference, you can use bower.json for most of the content here: https://github.com/aspnet/jquery-validation-unobtrusive/blob/master/bower.json